src/overlays/actors/ovl_Door_Shutter/z_door_shutter.c

```diff
@@ -896,9 +896,7 @@ void DoorShutter_PhantomGanonBarsRaise(DoorShutter* this, PlayState* play) {
     f32 targetOffsetY;
 
     PRINTF("FHG SAKU START !!\n");
-    if (this->isActive != 0) {
-        this->isActive--;
-    }
+    DECR(this->isActive);
```

Comment:
https://decomp.me/scratch/xDlhl

also matches with a `if(0){}` in a very specific spot https://decomp.me/scratch/E7oT2

Reply:
I think this is a reasonable match, DECR helped remove some `if(1) {}`s in EnNiw_Update too